python/rikai/types/vision.py

```diff
@@ -200,6 +200,11 @@ def to_numpy(self) -> np.ndarray:
         with self.to_pil() as pil_img:
             return np.asarray(pil_img)
 
+    def to_dict(self) -> dict:
+        if self.is_embedded:
```

Comment:
No strong opinion here. Just general question, why not return `{"data": self.data, "uri": null}` here?
Btw, should we do a base64 encoding on the data?

Reply:
yeah, added base64 encoding.
return schema: i don't have a strong preference either just wanted to avoid sending extra keys that we know are null given the context that it's table data so presumably the data schema is known/declared explicitly elsewhere

Reply:
Ok, that makes sense.